packages/kbn-docs-utils/src/api_docs/build_api_docs_cli.ts

```diff
@@ -130,7 +130,7 @@ export function runBuildApiDocsCli() {
         ]);
 
         const getLink = (d: ApiDeclaration) =>
-          `https://github.com/elastic/kibana/tree/master/${d.source.path}#L${d.source.lineNumber}`;
+          `https://github.com/elastic/kibana/tree/master/${d.path}#:~:text=${d.label}`;
```

Comment:
I think we might want to wrap the `d.label` in an `encodeURIComponent` to make sure that some special chars in type names don't break with the URL.